histoqc/SaveModule.py

```diff
@@ -46,21 +48,23 @@ def saveAssociatedImage(s, key:str, dim:int):
         s["warnings"].append(message)
         return
     
-    # get asscociated image by key
+    # get associated image by key
     associated_img = osh.associated_images[key]
-    (width, height)  = associated_img.size
+    associated_img = np.asarray(associated_img)[:, :, 0:3]
+    (height, width, _)  = associated_img.shape
 
-    # calulate the width or height depends on dim
+    # calculate the width or height depending on dim
     if width > height:
         h = round(dim * height / width)
         size = (dim, h)
     else:
         w = round(dim * width / height)
         size = (w, dim)
     
-    associated_img = associated_img.resize(size)
-    associated_img = np.asarray(associated_img)[:, :, 0:3]
-    io.imsave(f"{s['outdir']}{os.sep}{s['filename']}_{key}.png", associated_img)
+    associated_img = transform.resize(associated_img, size)
+    ext = os.path.splitext(s['filename'])[1]
```

Comment:
Hi @ajinkya-kulkarni , **s['filename']** has the current slide image name such as xxx-xxx-xxx.svs so the file ext is **.svs** not **.png**. We want the output image in PNG format.